rollup.config.js

```diff
@@ -0,0 +1,9 @@
+import ts from '@wessberg/rollup-plugin-ts'
+
+export default {
+  plugins: [
```

Comment:
I think you might be missing [`@rollup/plugin-node-resolve`](https://github.com/rollup/plugins/tree/master/packages/node-resolve), configured with `extensions: ['js', 'ts']`?